### ISSUE TITLE
[refactoring] refactor all error messages, form text into a json file 

### ISSUE THREAD

**Issue by roshaans** [near/ui.multisafe#150]
Similarly to[ Near-wallet ](https://github.com/near/near-wallet/tree/master/packages/frontend/src/translations), it would be helpful to refactor the app so that all form text, error messages, and descriptive text are documented and sourced from a single json file. 

This will help us to keep a common language in the app and keep us from worrying about having to change a given text in all parts of the application as well as help us roll out translations further down the road. 

brought up by @Patrick1904 in https://github.com/near/ui.multisafe/pull/147#discussion_r933575870_

**Comment by Patrick1904** [comment]
Yes we might as well add necessary framework to support multiple languages should it be needed in the future. The NEAR Wallet currently uses `react-localize-redux`, however, the package is barely maintained anymore and I suggest we research a better option.
